### PR TITLE
AlertContainer, AlertMeetingContainer, and AlertCallContainer Implementations

### PR DESCRIPTION
**Pull request by jlowery457**
screen shot 2018-06-01 at 2 20 32 pm
screen shot 2018-06-01 at 2 30 26 pm
screen shot 2018-06-01 at 3 36 18 pm

### CODE REVIEW

File: src/lib/AlertMeetingContainer/index.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { AlertMeeting } from '@collab-ui/react';
+import {
+  bind,
+  reject,
+  uniqueId
+} from 'lodash';
+
+class AlertMeetingContainer extends React.Component {
```
Comment (shashankcs090) [inline src/lib/AlertMeetingContainer/index.js]: Had a question from usage point of view, Did you explore using context to expose these functions outside?

File: src/lib/AlertContainer/index.js [hunk 2]
```diff
@@ -1,37 +1,117 @@
...
+        }
+      </div>
+    );
+  }
+}
 
 AlertContainer.defaultProps = {
-  children: null,
-  className: '',
+  newestOnTop: true,
```
Comment (bfbiggs) [inline src/lib/AlertContainer/index.js]: perhaps to remove the preposition/3 words we could call it orderNewest? @pauljeter

File: src/lib/AlertMeetingContainer/index.js (hunk 1 above)
Comment (bfbiggs) [reply src/lib/AlertMeetingContainer/index.js]: Might be something to look at for phase 2

File: src/lib/AlertMeetingContainer/index.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  static displayName = 'AlertMeetingContainer';
+
+  state = {
+    alertList: []
+  }
+
+  meetingAlert(title, status, message, onHide, onSnooze, attendees, avatar) {
```
Comment (bfbiggs) [inline src/lib/AlertMeetingContainer/index.js]: arrow functions

**Pull request merged**